### PR TITLE
Feature/insert datetime

### PR DESCRIPTION
**Pull request by PGilbertSchmitt**
Started the Datetime functionality. Will start on modifying datetime in-place

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 out
-node_modules
\ No newline at end of file
+node_modules
+src/*.js
```
Comment (jsonreeder) [inline .gitignore]: Why is this necessary? I don't see any `.js` files in `src/`.

File: package.json [hunk 2]
```diff
@@ -23,6 +23,10 @@
             { 
                 "command": "extension.insertChild",
                 "title": "Org-mode: Insert Child"
+            },
+            {
+                "command": "extension.insertDate",
+                "title": "Org-mode: Insert Date"
```
Comment (jsonreeder) [inline package.json]: Mind using the title from `feature_queue.org`: "Org: Time Stamp"? The official org name is `org-time-stamp`. I'm down for either capitals with spaces or kebab case.

File: .gitignore (hunk 1 above)
Comment (PGilbertSchmitt) [inline .gitignore]: This is just for any files I create while I'm doing a small test. If I compile a typescript file in that directory, it won't be commited
